### DISCUSSION TITLE
Can not uppload files to github

### DISCUSSION THREAD

**Discussion by Nazira20** [community/community#63246] [New to GitHub]
It appears an error when I try to upload files to github: "ImportError: cannot import name 'make_generic_validator' from 'pydantic.class_validators' (/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/pydantic/class_validators.py)"
Do anyone know how to fix this issue?

**Comment by nirav-gajera** [comment]
The error message you are getting is saying that the `make_generic_validator` function cannot be imported from the `pydantic.class_validators` module. This is likely because you are using an older version of Pydantic that does not have this function.

To fix this issue, you need to update your version of Pydantic. You can do this by running the following command in your terminal:

```
pip install pydantic --upgrade
```

Once you have updated your version of Pydantic, you should be able to import the `make_generic_validator` function and fix the error.

Here are some additional resources that you may find helpful:

* Pydantic documentation: https://pydantic-docs.helpmanual.io/
* Pydantic GitHub repository: https://github.com/samuelcolvin/pydantic
